documentation/airport-guides/rjoa.md

```diff
@@ -0,0 +1,55 @@
+# RJOA - Hiroshima Airport
+
+## About this facility
+
+Hiroshima Airport is a Japanese airport in the city of Mihara, Hiroshima Prefecture. It is located approximately 50km east from the center of Hiroshima City.
+
+Opened in 1993, it is now the largest airport in the Chūgoku region, serving 2.7 million passengers domestically and internationally in 2014.
+
+It is built on a mound formed by piling up soil from mountains around it, making the elevation high, at around 1000ft. Due to its high elevation, sudden wind changes and foggy weathers are quite common. To prevent flight cancellations and diversions, the airport authority installed the CAT IIb ILS in 2009 for RWY10. ILS is not installed for RWY28.
+
+## Positions
+
+|Callsign             | Frequency  | Remarks
+|---------------------|:----------:|:---------------------------------------------:|
+|"HIROSHIMA TOWER"    | 118.600MHz |                                               |
+|                     | 126.200MHz |                                               |
+|"HIROSHIMA DEPARTURE"| 119.900MHz |                                               |
+|"HIROSHIMA APPROACH" | 124.050MHz |                                               |
+|                     | 119.900MHz |                                               |
+|"HIROSHIMA RADAR"    | 119.900MHz |                                               |
+|                     | 124.050MHz |                                               |
+|                     | 125.150MHz |                                               |
+|"HIROSHIMA ATIS"     | 127.250MHz |                                               |
+
+## Airspace
+
+There are four radar positions in the Hiroshima Approach Control Area (ACA) — Depature, Approach, and Radar.
+
+As the name suggests, the Departure and Approach positions are for departing and arriving aircrafts to Sendai Airport.
+
+The Radar position is used as a combined position of Depature and Approach.
+
+## Instrument Procedures
+###Departures
+
+Departing aircraft will be assigned the MOMOT THREE Depature, or KIJYY TWO Departure in the simulation; they are generally assigned to aircraft departing north/eastbound and west/southbound respectively. Pilots are given an initial climb of FL160. 
+
+###Arrivals
+After handed off from Control, Hiroshima Approach is responsible for: 
+* issuing further descent clearance;
+* vectoring the aircraft to the final approach course if necessary;
+* issuing approach clearance
+* ensuring sufficient spacing between aircraft;
+
+MISEN is the IF (Intermediate Fix), and MACKY the FAF (Final Approach Fix) for ILS 10 approach which should be used in the scenario. Approach should issue a descent clearance so that the aircraft is at 3600ft before reaching YUCCA.
```

Comment:
I assume that should say MACKY, not YUCCA. It'd also be good to have MACKY in the fixes section.